### PR TITLE
Add google site

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Added London google site page
- Updated index page

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1563"></a>

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Comment by christopher-hakkaart** [comment]
Will wait for check from @drpatelh before merge 👀

**Comment by maxulysse** [comment]
why has the `package-lock.json` file been updated?

**Comment by christopher-hakkaart** [comment]
> why has the `package-lock.json` file been updated?

No idea... Something weird happened on my end. I'll revert it now.

File: markdown/events/2023/hackathon-march-2023/uk-google.md
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:12]:
```suggestion
address: 123 Buckingham Palace Rd, London SW1W 9SH
location_url: https://www.wired.co.uk/article/google-digital-skills-academy
location_latlng: [51.493721, -0.146830]
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:3]:
```suggestion
subtitle: Local node of the nf-core Hackathon at Google Academy, London.
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md
Comment (drpatelh) [reply markdown/events/2023/hackathon-march-2023/uk-google.md:12]: Thanks @christopher-hakkaart! Can you just check the `location_lating` entry looks good? Otherwise good to merge.

**Pull request merged**
